resources/translations/en.edn

```diff
@@ -265,8 +265,8 @@
   :link {:download-file "Download file"
          :opens-in-new-window "Opens in new window"}
   :login {:alternative "Other sign in options"
-          :intro [:div [:h1 "Welcome to REMS"] [:p "This is a demo environment for testing the REMS software."]
-                  [:p "Please, login first to access REMS."]]
+          :intro [:div [:h1 "Welcome to REMS"] [:p "This is a demo environment for testing the REMS software. The demo environment has a couple of fictional datasets to which you can apply for access. More information available on the " [:a {:href "#/extra-pages/about"} "About"] " page."]
```

Comment:
A proposal: "More information available on the About page." -> "More information is available on About page." (The name of the page is About, so I believe it doesn't need the article. Also added a verb to the sentence.)